powerhome/playbook
# [PLAY-312] - Remove Rails Visual Guidelines Code and Replace with React

#### Screens
![Screen Shot 2022-09-13 at 4 21 23 PM](https://user-images.githubusercontent.com/73710701/190002127-731fd03b-d241-49c6-862f-e4dc5bcfa879.png)

![Screen Shot 2022-09-13 at 4 21 32 PM](https://user-images.githubusercontent.com/73710701/190002142-065a2725-6f47-4f12-813c-477e79ef81c8.png)

fixed line height example:
![Screen Shot 2022-09-13 at 4 21 11 PM](https://user-images.githubusercontent.com/73710701/190002189-6c3f0737-03e7-422c-8fc6-7cfc5f0adc18.png)

fixed z-index example:
![Screen Shot 2022-09-13 at 4 20 58 PM](https://user-images.githubusercontent.com/73710701/190002234-106e9514-cc53-4b34-aef1-2941a1912d41.png)


#### Breaking Changes

No breaking changes, PR only effects visual guidelines page

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-312)

#### How to test this

Review in milano env. Navigate to /visual_guidelines page and make sure page renders correctly, sidebar menu functions as expected and  sidebar transitions to hamburger menu on smaller screens.

#### Checklist:
